internal/services/bot/bot_service_resource_base.go

```diff
@@ -107,6 +107,12 @@ func (br botBaseResource) arguments(fields map[string]*pluginsdk.Schema) map[str
 			}, false),
 		},
 
+		"local_auth_enabled": {
```

Comment:
this should be `local_authentication_enabled` to match other cases in the provider - can we update this?

Reply:
Updated